### PR TITLE
Doc typos 1

### PR DESCRIPTION
**Pull request by jtlap**
this correct most references links in non functions docs
also small typos
and try to unify vocabulary sometimes
only html are touched so cycle is quite meaningless, only the review counts

### CODE REVIEW

File: docs/tutorial/masking.html [hunk 1]
```diff
@@ -367,8 +367,8 @@
 </script>
 
 Conclusion
-====================================================================================================
```
Comment (jfalcou) [inline docs/tutorial/masking.html:370]: This is a marker to make a title, why do you remove it ?

File: docs/tutorial/strlen.html [hunk 2]
```diff
@@ -59,24 +68,24 @@
 `wide(ptr)` constructor does not have a way to do this.
 
 ### `cur == zeroes`
-This is a pairwise comparison for equality between `cur` and `zero`.
-The result is `eve::logical<wide>` - a class representing N booleans where N is the width of our wide.
+This is an elementwise comparison for equality between `cur` and `zeroes`.
+The result is `eve::logical<wide>` - a class representing N booleans where N is the cardinal (number of lanes) of our wide.
```
Comment (jfalcou) [inline docs/tutorial/strlen.html:72]: cardinal is enough

File: docs/tutorial/strlen.html [hunk 3]
```diff
@@ -59,24 +68,24 @@
...
 
 ### `eve::ignore_first ignore / eve::first_true[ignore]`
 `first_true` takes a `logical` and returns the optional position of the first `true` element.
-However we took a `previous_aligned_address` and not just `s` - so there might be some 0s we are not interested in. <br/>
+However we took a `previous_aligned_address(s)` and not just `s`, so there might be some 0s we are not interested in. <br/>
```
Comment (jfalcou) [inline docs/tutorial/strlen.html:76]: we took a -> we compute

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: docs/tutorial/strlen.html (hunk 2 above)
Comment (jtlap) [reply docs/tutorial/strlen.html:72]: c'est pour homogeneiser

File: docs/tutorial/strlen.html (hunk 3 above)
Comment (jtlap) [reply docs/tutorial/strlen.html:76]: done

File: docs/tutorial/masking.html (hunk 1 above)
Comment (jtlap) [reply docs/tutorial/masking.html:370]: pige pas pourquoi j'ai enlevé ça done

**Review by jfalcou — APPROVED** [review]

**Pull request merged**